wdl/subworkflows/get_first_read_in_pair_from_bam.wdl

```diff
@@ -0,0 +1,30 @@
+version 1.0
+
+
+import "../tasks/samtools.wdl"
+
+
+workflow get_first_read_in_pair_from_bam {
+    input {
+        File paired_end_bam
+        Int? first_in_pair_flag = 64
+        Resources resources
+        SamtoolsViewParams params = object {
+            include: first_in_pair_flag,
+            output_bam: true
+        }
```

Comment:
I don't quite understand this. Why is `first_in_pair_flag` declared as `Int?` despite it being defined as a `String?` in the underlying struct? Is there some reason to rely on implicit type conversion here? Couldn't it simply be defined as (nonoptional) `String` here? Interested to hear about this, because usually you have your reasons.

Reply:
No reason. Thanks for catching this.